modules/aioseop_opengraph.php

```diff
@@ -335,6 +335,10 @@ public function init() {
 					'name'     => __( 'Home Title', 'all-in-one-seo-pack' ),
 					'default'  => '',
 					'type'     => 'text',
+					'class'    => 'aioseop_count_chars',
+					'count'    => true,
+					'count_desc' => $count_desc,

```

Comment:
When you change the `=>` alignment, you usually want to align the rest. Otherwise WPCS would flag it.

For example

```
'hometitle'              => array(
	'name'       => __( 'Home Title', 'all-in-one-seo-pack' ),
	'default'    => '',
	'type'       => 'text',
	'class'      => 'aioseop_count_chars',
	'count'      => true,
	'count_desc' => $count_desc,
	'size'       => 95,
	'condshow'   => array(
		'aiosp_opengraph_setmeta' => array(
			'lhs' => 'aiosp_opengraph_setmeta',
			'op'  => '!=',
			'rhs' => 'on',
		),
	),
),
```

Reply:
Done.